Update README.md (#6223)
huggingface/datasets: docs/README.md

**NOTE**

You only need to generate the documentation to inspect it locally (if you're planning changes and want to

> check how they look before committing for instance). You don't have to commit to the built documentation.

Reviewer: Might be better to revert this here to avoid changing the meaning of it (`git commit` the generated docs versus commit to it).

```suggestion
check how they look before committing for instance). You don't have to `git commit` the built documentation.
```

---

## Building the documentation

> Once you have set the `doc-builder` and additional packages, you can generate the documentation by typing th
> the following command:

Reviewer: I think setup was correct the way before

```suggestion
Once you have setup the `doc-builder` and additional packages, you can generate the documentation by typing
the following command:
```

    - save_vocabulary
```

If you just want to add a method that is not documented (for instance magic method like `__call__` is not documented

> by default) you can add the list of methods to add in a list that contains `all`:

Reviewer: I think it sounded better before to avoid using two `add`'s in the same sentence

```suggestion
by default) you can put the list of methods to add in a list that contains `all`:
```
